work/worker.go

```diff
@@ -774,6 +779,7 @@ func (env *Task) ApplyTransactions(txs *types.TransactionsByPriceAndNonce, bc Bl
 	var numTxsNonceTooLow int64 = 0
 	var numTxsNonceTooHigh int64 = 0
 	var numTxsGasLimitReached int64 = 0
+	txAbortedByTimeLimit := false // indicate that first tx is elapsed over execution time budget
```

Comment:
Isn't it the opposite?

txAbortedByTimeLimit // is true when a transaction was aborted mid-execution due to time limit. Note that the first transaction or first bundle of a block does not abort.

Reply:
Ahh, right. I was too biased and focused on a `false` value. Your description is correct. Fixed.